grails-app/taglib/grails/plugin/cache/CacheTagLib.groovy

```diff
@@ -82,34 +89,38 @@ class CacheTagLib {
 	 */
 	def render = { attrs ->
 		if (!grailsCacheManager) {
-			out <<  g.render(attrs)
+			out << g.render(attrs)
 			return
 		}
 
-		//Make this empty string to save error later in grails gsp core
-		attrs.plugin = attrs.plugin ?: ''
+		try {
```

Comment:
Adding try catch here and falling back to attempt to render the template on error